src/SonarQube.Client/Api/Requests/V7_20/GetIssuesRequest.cs

```diff
@@ -86,22 +86,20 @@ private static SonarQubeIssueResolutionState ParseResolutionState(string resolut
 
         private class ServerIssue
         {
-            [JsonProperty("key")]
-            public string Key { get; set; }
+            [JsonProperty("rule")]
+            public string CompositeRuleKey { get; set; }
             [JsonProperty("component")]
             public string Component { get; set; }
+            [JsonProperty("subProject")]
+            public string SubProject { get; set; }
             [JsonProperty("hash")]
             public string Hash { get; set; }
             [JsonProperty("line")]
-            public int Line { get; set; }
+            public int? Line { get; set; }
```

Comment:
Changed into nullable because module level issues don't have a line (before it was defaulting to default(int)).